.github/workflows/maven-verify.yml

```diff
@@ -184,10 +184,17 @@ jobs:
 
       - name: Build with Maven
         run: ./mvnw --errors --batch-mode --show-version ${{ inputs.maven-args }} ${{ inputs.ff-goal }}
+        
+      - name: Clean Ensuring no file handle remains open on windows
+        run: ./mvnw clean --errors --batch-mode --show-version           
 
       - name: Build Maven Site
         run: ./mvnw --errors --batch-mode --show-version ${{ inputs.maven-args }} ${{ inputs.ff-site-goal }}
         if: inputs.ff-site-run
+        
+      - name: Clean Ensuring no file handle remains open on windows
+        if: steps.should-run.conclusion == 'success'
+        run: ./mvnw clean --errors --batch-mode --show-version            
```

Comment:
put it after `Show free disk space` we need files for `Upload Maven Site` and for `Upload artifact on failure`

Reply:
yup we don't know if the basic build or the site build is locking.
but that would be nit picking